qlib/data/ops.py

```diff
@@ -1569,7 +1569,6 @@ def _load_internal(self, instrument, start_index, end_index, *args):
 
 TOpsList = [TResample]
 OpsList = [
-    ChangeInstrument,
```

Comment:
Hi, Why do you remove this operator?

Reply:
Oh, I didn't mean to remove it. I'll fix it. Thanks!